src/Windowing/OpenToolkit.Windowing.Common/Interfaces/INativeWindow.cs

```diff
@@ -53,5 +53,11 @@ public interface INativeWindow : INativeWindowProperties, IWindowEvents, IWindow
         /// The point transformed to screen coordinates.
         /// </returns>
         Vector2i PointToScreen(Vector2i point);
+
+        /// <summary>
+        /// Gets the current monitor's pixel-per-inch count.
+        /// </summary>
+        /// <returns>The current monitor ppi.</returns>
+        int CurrentMonitorPpi { get; }
```

Comment:
If PointsPerInch (ppi) is the same as DotsPerInch (dpi) then I'd rather have these named CurrentMonitorDpi

Reply:
PPI seems to be more accurate but we should probably copy what other apis do